content/guides/references/advanced-installation.md

```diff
@@ -244,6 +244,15 @@ To save the `CYPRESS_CRASH_REPORTS` variable for use in all new shells, use
 setx CYPRESS_CRASH_REPORTS 0
 ```
 
+## Opt out of Cypress commercial messaging
+
+Cypress may occasionally display messages related to our commercial offerings
+and how they could benefit you during your workflows.
```

Comment:
I think it would be helpful to clarify where exactly these messages will appear. As a user, if I think these commercial messages will be turning in up in e.g. my CI logs, I'm more inclined to turn them off. Whereas if they only appear in the runner, I probably wouldn't bother.